### ISSUE TITLE
训练时候出错

### ISSUE THREAD

**Issue by Sun-Shun** [PaddlePaddle/PaddleClas#1345]
欢迎您使用PaddleClas并反馈相关问题，非常感谢您对PaddleClas的贡献！
提出issue时，辛苦您提供以下信息，方便我们快速定位问题并及时有效地解决您的问题：
 1. PaddleClas版本以及PaddlePaddle版本：请您提供您使用的版本号或分支信息，如PaddleClas release/2.2和PaddlePaddle 2.1.0
 2. 涉及的其他产品使用的版本号：如您在使用PaddleClas的同时还在使用其他产品，如PaddleServing、PaddleInference等，请您提供其版本号
 3. 训练环境信息：
  a. 具体操作系统，如Linux/Windows/MacOS
  b. Python版本号，如Python3.6/7/8
  c. CUDA/cuDNN版本， 如CUDA10.2/cuDNN 7.6.5等
 4. 完整的代码(相比于repo中代码，有改动的地方)、详细的错误信息及相关log

训练指令：python  tools/train.py -c ./ppcls/configs/quick_start/professional/ResNet50_vd_CIFAR100.yaml  -o Global.output_dir="output_CIFAR"  -o Arch.pretrained=True

训练显示结果，但一直报错！！

C:\Users\86151/.paddleclas/weights\ResNet50_vd_pretrained.pdparams
C:\Users\86151\Anaconda3\lib\site-packages\paddle\fluid\dygraph\layers.py:1301: UserWarning: Skip loading for fc.weight. fc.weight receives a shape [2048, 1000], but the expected shape is [2048, 12].
  warnings.warn(("Skip loading for {}. ".format(key) + str(err)))
C:\Users\86151\Anaconda3\lib\site-packages\paddle\fluid\dygraph\layers.py:1301: UserWarning: Skip loading for fc.bias. fc.bias receives a shape [1000], but the expected shape is [12].
  warnings.warn(("Skip loading for {}. ".format(key) + str(err)))
[2021/10/27 15:51:29] root INFO: train with paddle 2.1.3 and device CUDAPlace(0)
C:\Users\86151\Anaconda3\lib\site-packages\paddle\fluid\reader.py:358: UserWarning: DataLoader with multi-process mode is not supported on MacOs and Windows currently. Please use signle-process mode with num_workers = 0 instead
  "DataLoader with multi-process mode is not supported on MacOs and Windows currently." \
[2021/10/27 15:51:30] root INFO: [Train][Epoch 1/300][Iter: 0/34]lr: 0.04000, CELoss: 2.48894, loss: 2.48894, top1: 0.03125, top5: 0.48438, batch_cost: 0.87071s, reader_cost: 0.15462, ips: 73.50348 images/sec, eta: 2:28:01
[2021/10/27 15:51:30] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/tO6cKGH8uPEayzmeZJ51Fdr2Tx3fBYSn.jpg with msg: 'NoneType' object has no attribute 'shape'
[2021/10/27 15:51:30] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/YfsxcFB9D3LvkdQyiXlqnNZ4STwope2r.jpg with msg: 'NoneType' object has no attribute 'shape'
[2021/10/27 15:51:31] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/yGcJHV8Uuft6grFs7QWnK5CTAZvYzdDO.jpg with msg: 'NoneType' object has no attribute 'shape'
[2021/10/27 15:51:31] root INFO: [Train][Epoch 1/300][Iter: 10/34]lr: 0.04000, CELoss: 2.60209, loss: 2.60209, top1: 0.09375, top5: 0.46023, batch_cost: 0.12350s, reader_cost: 0.00682, ips: 518.20466 images/sec, eta: 0:20:58
[2021/10/27 15:51:31] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/3yMZzWekKmuoGOF60ICQxldhBEc9Ra15.jpg with msg: 'NoneType' object has no attribute 'shape'
[2021/10/27 15:51:32] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/5nKsehtjrXCZqbAcSW13gxB8E6z2Luy7.jpg with msg: 'NoneType' object has no attribute 'shape'
[2021/10/27 15:51:33] root INFO: [Train][Epoch 1/300][Iter: 20/34]lr: 0.04000, CELoss: 2.55338, loss: 2.55338, top1: 0.11905, top5: 0.50074, batch_cost: 0.16357s, reader_cost: 0.04662, ips: 391.27521 images/sec, eta: 0:27:45
[2021/10/27 15:51:35] root INFO: [Train][Epoch 1/300][Iter: 30/34]lr: 0.04000, CELoss: 2.52793, loss: 2.52793, top1: 0.12349, top5: 0.50907, batch_cost: 0.17201s, reader_cost: 0.05527, ips: 372.08184 images/sec, eta: 0:29:09
[2021/10/27 15:51:35] root INFO: [Train][Epoch 1/300][Avg]CELoss: 2.52594, loss: 2.52594, top1: 0.12500, top5: 0.51468
[2021/10/27 15:51:35] root INFO: [Train][Epoch 2/300][Iter: 0/34]lr: 0.04000, CELoss: 2.40325, loss: 2.40325, top1: 0.10938, top5: 0.59375, batch_cost: 0.17696s, reader_cost: 0.06030, ips: 361.67293 images/sec, eta: 0:29:58
[2021/10/27 15:51:37] root INFO: [Train][Epoch 2/300][Iter: 10/34]lr: 0.04000, CELoss: 2.35911, loss: 2.35911, top1: 0.17898, top5: 0.60511, batch_cost: 0.17437s, reader_cost: 0.05785, ips: 367.02882 images/sec, eta: 0:29:30
[2021/10/27 15:51:38] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/3yMZzWekKmuoGOF60ICQxldhBEc9Ra15.jpg with msg: 'NoneType' object has no attribute 'shape'
[2021/10/27 15:51:39] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/YfsxcFB9D3LvkdQyiXlqnNZ4STwope2r.jpg with msg: 'NoneType' object has no attribute 'shape'
[2021/10/27 15:51:39] root INFO: [Train][Epoch 2/300][Iter: 20/34]lr: 0.04000, CELoss: 2.34280, loss: 2.34280, top1: 0.18378, top5: 0.61607, batch_cost: 0.18401s, reader_cost: 0.06794, ips: 347.80181 images/sec, eta: 0:31:06

**Comment by trarynight** [comment]
> 欢迎您使用PaddleClas并反馈相关问题，非常感谢您对PaddleClas的贡献！ 提出issue时，辛苦您提供以下信息，方便我们快速定位问题并及时有效地解决您的问题：
> 
> 1. PaddleClas版本以及PaddlePaddle版本：请您提供您使用的版本号或分支信息，如PaddleClas release/2.2和PaddlePaddle 2.1.0
> 2. 涉及的其他产品使用的版本号：如您在使用PaddleClas的同时还在使用其他产品，如PaddleServing、PaddleInference等，请您提供其版本号
> 3. 训练环境信息：
>    a. 具体操作系统，如Linux/Windows/MacOS
>    b. Python版本号，如Python3.6/7/8
>    c. CUDA/cuDNN版本， 如CUDA10.2/cuDNN 7.6.5等
> 4. 完整的代码(相比于repo中代码，有改动的地方)、详细的错误信息及相关log
> 
> 训练指令：python tools/train.py -c ./ppcls/configs/quick_start/professional/ResNet50_vd_CIFAR100.yaml -o Global.output_dir="output_CIFAR" -o Arch.pretrained=True
> 
> 训练显示结果，但一直报错！！
> 
> C:\Users\86151/.paddleclas/weights\ResNet50_vd_pretrained.pdparams C:\Users\86151\Anaconda3\lib\site-packages\paddle\fluid\dygraph\layers.py:1301: UserWarning: Skip loading for fc.weight. fc.weight receives a shape [2048, 1000], but the expected shape is [2048, 12]. warnings.warn(("Skip loading for {}. ".format(key) + str(err))) C:\Users\86151\Anaconda3\lib\site-packages\paddle\fluid\dygraph\layers.py:1301: UserWarning: Skip loading for fc.bias. fc.bias receives a shape [1000], but the expected shape is [12]. warnings.warn(("Skip loading for {}. ".format(key) + str(err))) [2021/10/27 15:51:29] root INFO: train with paddle 2.1.3 and device CUDAPlace(0) C:\Users\86151\Anaconda3\lib\site-packages\paddle\fluid\reader.py:358: UserWarning: DataLoader with multi-process mode is not supported on MacOs and Windows currently. Please use signle-process mode with num_workers = 0 instead "DataLoader with multi-process mode is not supported on MacOs and Windows currently." [2021/10/27 15:51:30] root INFO: [Train][Epoch 1/300][Iter: 0/34]lr: 0.04000, CELoss: 2.48894, loss: 2.48894, top1: 0.03125, top5: 0.48438, batch_cost: 0.87071s, reader_cost: 0.15462, ips: 73.50348 images/sec, eta: 2:28:01 [2021/10/27 15:51:30] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/tO6cKGH8uPEayzmeZJ51Fdr2Tx3fBYSn.jpg with msg: 'NoneType' object has no attribute 'shape' [2021/10/27 15:51:30] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/YfsxcFB9D3LvkdQyiXlqnNZ4STwope2r.jpg with msg: 'NoneType' object has no attribute 'shape' [2021/10/27 15:51:31] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/yGcJHV8Uuft6grFs7QWnK5CTAZvYzdDO.jpg with msg: 'NoneType' object has no attribute 'shape' [2021/10/27 15:51:31] root INFO: [Train][Epoch 1/300][Iter: 10/34]lr: 0.04000, CELoss: 2.60209, loss: 2.60209, top1: 0.09375, top5: 0.46023, batch_cost: 0.12350s, reader_cost: 0.00682, ips: 518.20466 images/sec, eta: 0:20:58 [2021/10/27 15:51:31] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/3yMZzWekKmuoGOF60ICQxldhBEc9Ra15.jpg with msg: 'NoneType' object has no attribute 'shape' [2021/10/27 15:51:32] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/5nKsehtjrXCZqbAcSW13gxB8E6z2Luy7.jpg with msg: 'NoneType' object has no attribute 'shape' [2021/10/27 15:51:33] root INFO: [Train][Epoch 1/300][Iter: 20/34]lr: 0.04000, CELoss: 2.55338, loss: 2.55338, top1: 0.11905, top5: 0.50074, batch_cost: 0.16357s, reader_cost: 0.04662, ips: 391.27521 images/sec, eta: 0:27:45 [2021/10/27 15:51:35] root INFO: [Train][Epoch 1/300][Iter: 30/34]lr: 0.04000, CELoss: 2.52793, loss: 2.52793, top1: 0.12349, top5: 0.50907, batch_cost: 0.17201s, reader_cost: 0.05527, ips: 372.08184 images/sec, eta: 0:29:09 [2021/10/27 15:51:35] root INFO: [Train][Epoch 1/300][Avg]CELoss: 2.52594, loss: 2.52594, top1: 0.12500, top5: 0.51468 [2021/10/27 15:51:35] root INFO: [Train][Epoch 2/300][Iter: 0/34]lr: 0.04000, CELoss: 2.40325, loss: 2.40325, top1: 0.10938, top5: 0.59375, batch_cost: 0.17696s, reader_cost: 0.06030, ips: 361.67293 images/sec, eta: 0:29:58 [2021/10/27 15:51:37] root INFO: [Train][Epoch 2/300][Iter: 10/34]lr: 0.04000, CELoss: 2.35911, loss: 2.35911, top1: 0.17898, top5: 0.60511, batch_cost: 0.17437s, reader_cost: 0.05785, ips: 367.02882 images/sec, eta: 0:29:30 [2021/10/27 15:51:38] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/3yMZzWekKmuoGOF60ICQxldhBEc9Ra15.jpg with msg: 'NoneType' object has no attribute 'shape' [2021/10/27 15:51:39] root ERROR: Exception occured when parse line: ./dataset/Cat/cat_12_train/YfsxcFB9D3LvkdQyiXlqnNZ4STwope2r.jpg with msg: 'NoneType' object has no attribute 'shape' [2021/10/27 15:51:39] root INFO: [Train][Epoch 2/300][Iter: 20/34]lr: 0.04000, CELoss: 2.34280, loss: 2.34280, top1: 0.18378, top5: 0.61607, batch_cost: 0.18401s, reader_cost: 0.06794, ips: 347.80181 images/sec, eta: 0:31:06

I got same error， and i found my config file have some wrong setting in transform. plz double check your config file.

**Comment by cuicheng01** [comment]
There may be a problem with the image, you can check whether the image format is correct, such as this image:`./dataset/Cat/cat_12_train/tO6cKGH8uPEayzmeZJ51Fdr2Tx3fBYSn.jpg`

**Comment by Sun-Shun** [comment]
> There may be a problem with the image, you can check whether the image format is correct, such as this image:`./dataset/Cat/cat_12_train/tO6cKGH8uPEayzmeZJ51Fdr2Tx3fBYSn.jpg`

检测图片路径完全没有任何问题

**Comment by Sun-Shun** [comment]
> There may be a problem with the image, you can check whether the image format is correct, such as this image:`./dataset/Cat/cat_12_train/tO6cKGH8uPEayzmeZJ51Fdr2Tx3fBYSn.jpg`

there is no problems

**Comment by cuicheng01** [comment]
图片方便发一下吧，我们来看下

**Comment by lazyn1997** [comment]
我在设置非方形缩放时也出现了这个错误具体报错如下：
 root ERROR: Exception occured when parse line: G:/postgraduate_data/feature/LYC_filter/167/167_second_train_20_1.png with msg: operands could not be broadcast together with shapes (1120,448,3) (448,1120,3) (1120,448,3)
请问应该如何解决，检查了图像路径是没有问题的

**Comment by aiot-tech** [comment]
met the same problem

**Comment by aiot-tech** [comment]
but after correcting the img size in the yaml,the problem solved

**Comment by ChrisMengxl** [comment]
> 图片方便发一下吧，我们来看下

我也遇到同样的问题，在AiStudio训练，方便复现。
root ERROR: Exception occured when parse line: /home/aistudio/data/XXX/sun_akbocuwclkxqlofx.jpg with msg: 'NoneType' object has no attribute 'shape
